R/git.R

```diff
@@ -85,6 +85,22 @@ git_fetch <- function(root = NULL) {
 }
 
 git_pull <- function(root = NULL) {
-  orderly_log("git", "fetch")
+  orderly_log("git", "pull")
   git_run("pull", root = root, check = TRUE)
 }
+
+git_branches_no_merged <- function(root = NULL, include_master = FALSE) {
```

Comment:
When do we not want to include master?

Reply:
Never I don't think. Put this in as having `git_branches_no_merged` return master is different to how git behaves with `--no-merged` arg. So putting it as an arg was just to make it explicit. It also works slightly differently to the other branches as it is always first in the list. Happy to remove that though and return the branches we want.